include/cadmium/engine/pdevs_simulator.hpp

```diff
@@ -74,13 +76,24 @@ namespace cadmium {
              * @param initial_time is the start time
              */
             void init(TIME initial_time) {
-                LOGGER::template log<cadmium::logger::logger_info, decltype(formatter::log_info_init), TIME>(formatter::log_info_init, initial_time);
+
+                //logging data
+                std::ostringstream oss;
```

Comment:
Similar as in coordinator comments. May be better to wrap this into the formatter and maybe pass the type as parameter of the formatter funciton. Reason is avoiding dependency and doing this work when not used.

Reply:
here I will argue as in coordinator, I would keep the model type outside the formatter scope for performance purposes and to have a cleanest code.